packages/auto-label/README.md

```diff
@@ -4,11 +4,11 @@ The auto-label bot automatically label issues and pull requests.
 
 There are 3 types of labels available:
 1. Product labels (enabled by default): Product-specific repos are always labeled with the API label
-for that product. The product can also be auto-detected from the issue/PR title.
+for that product. The product can also be auto-detected from the issue/PR title. If you do not want an issue to have its api auto-detected by auto-label, simply label the issue with an "api: N/A" label.
```

Comment:
I always feel like an idiot when instructions tell me to 'simply do yadayada' and I find it difficult.  My $0.02 is to remove the word 'simply' from this sentence.

Reply:
will do!